utils/simulations_v2_1.csv

```diff
@@ -1,10 +1,13 @@
-model_version, group, resolution, category, simulation_name,    machine, checksum, experiment, ensemble_num, cmip_only, node,
-v2.1, WaterCycle, LR, DECK,       v2_1.LR.piControl,            chrysalis, ebc5565e182e8b1cc887c185ca3eb144, piControl,    1, cmip_only, cels.anl,
-v2.1, WaterCycle, LR, DECK,       v2_1.LR.1pctCO2_0101,         chrysalis, b7676ccc791941d1bdadde05c59d70b0, 1pctCO2,      1, cmip_only, cels.anl,
-v2.1, WaterCycle, LR, DECK,       v2_1.LR.abrupt-4xCO2_0101,    chrysalis, fe50274534774678d55cb320412d1667, abrupt-4xCO2, 1, cmip_only, cels.anl,
-v2.1, WaterCycle, LR, DECK,       v2_1.LR.amip_0101,            chrysalis, 832ca9c9db8b4988a109f8c5872b0fe7, amip,         1, cmip_only, cels.anl,
-v2.1, WaterCycle, LR, Historical, v2_1.LR.historical_0101,      chrysalis, 43efe5ba9216b535be21bd8474b15c22, historical,   1, cmip_only, cels.anl,
-v2.1, WaterCycle, LR, Historical, v2_1.LR.historical_0151,      chrysalis,                                 , historical,   2, cmip_only, cels.anl,
-v2.1, WaterCycle, LR, Historical, v2_1.LR.historical_0201,      chrysalis,                                 , historical,   3, cmip_only, cels.anl,
-v2.1, WaterCycle, LR, Historical, v2_1.LR.historical_0251,      chrysalis,                                 , historical,   4, cmip_only, cels.anl,
-v2.1, WaterCycle, LR, Historical, v2_1.LR.historical_0301,      chrysalis,                                 , historical,   5, cmip_only, cels.anl,
+model_version, group, resolution, category, simulation_name, machine, checksum, experiment, ensemble_num, cmip_only, node,
```

Comment:
5. checksums -- do these simulations have checksums computed? As in https://acme-climate.atlassian.net/wiki/spaces/EWCG/pages/3051357093/v2.LR.hist-GHG#Sanity-checks%3A-0101-%3Acheck_mark%3A for example.

(The other empty columns are only relevant if we want ESGF links.)